### DISCUSSION TITLE
VcLayerImagery裡放置VcImageryProviderWmts 調整對比,亮度無功能

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#354] [Q&A]
您好,
VcLayerImagery裡放置VcImageryProviderWmts 調整VcLayerImagery裡的對比,亮度 無功能
但是使用您的示例是沒問題
這是圖資來源的問題?
謝謝

**Comment by stan88** [comment]
抱歉,
這問題應該是VcImageryProviderWmts 是無法顯示影像圖
我是透過new Cesium.WebMapTileServiceImageryProvider添加地圖
是不是不屬於VC的部份是無法控制的?

**Reply by zouyaoji** [reply]
类似用 new Cesium.WebMapTileServiceImageryProvider 等Cesium api 初始化的实例，需要你自己管理。

**Reply by stan88** [reply]
己解決,謝謝
